### PR TITLE
Support concurrent web app deploy effectors

### PR DESCRIPTION
**Pull request by neykov**

### CODE REVIEW

File: software/webapp/src/test/java/brooklyn/entity/webapp/WebAppConcurrentDeployTest.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import brooklyn.test.entity.TestApplication;
+import brooklyn.util.collections.MutableList;
+import brooklyn.util.collections.MutableMap;
+import brooklyn.util.http.HttpTool;
+import brooklyn.util.http.HttpToolResponse;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+public class WebAppConcurrentDeployTest {
```
Comment (aledsage) [inline software/webapp/src/test/java/brooklyn/entity/webapp/WebAppConcurrentDeployTest.java]: Could this instead extend `BrooklynAppUnitTestSupport`?

**Comment by aledsage** [comment]
Looks good; only one very minor comment about the test. Happy for this to be merged.

**Comment by neykov** [comment]
Addressed comments (along with the comments from #609), thanks @aledsage

**Comment by neykov** [comment]
Any more comments @aledsage , can this be merged?

**Comment by aledsage** [comment]
Thanks @neykov - merging.

**Pull request merged**